cms/templates/js/login.html

```diff
@@ -0,0 +1,17 @@
+<div class="wrapper-content wrapper">
```

Comment:
it looks like you should probably put this in the mock folder in cms/templates/js/